web/styles/zulip.css

```diff
@@ -3295,3 +3295,86 @@ select.invite-as {
         padding: 5px;
     }
 }
+
+#add-poll-modal {
+    /* on wide screens, this height allows roughly 3 option
+    rows to fit in without need for scrolling */
+    height: 450px;
+    overflow: hidden;
+
+    @media (width < $sm_min) {
+        height: 80vh;
+    }
+
+    .modal__content {
+        flex-grow: 1;
```

Comment:
I can't see how this has an effect on anything in the modal, but then I think the use of `flex-direction: column` on most modals is unnecessary.

Reply:
Right, this is redundant. Removed this rule, thanks for pointing this out!